source/iface/context.rst

```diff
@@ -135,9 +136,12 @@ Returns information about the context as determined by ``param``. See
 ::
 
   enum class context : int {
-    reference_count,
     platform,
-    devices
+    devices,
+    atomic_memory_order_capabilities,
+    atomic_fence_order_capabilities,
+    atomic_memory_scope_capabilities,
+    atomic_fence_scope_capabilities
   };
```

Comment:
The information descriptors are not an enumeration.  (This was one of the changes we made since 1.2.1.)  Instead, they are types declared like:

```
namespace sycl::info::context {

struct platform;
struct devices;
struct atomic_memory_order_capabilities;
struct atomic_fence_order_capabilities;
struct atomic_memory_scope_capabilities;
struct atomic_fence_scope_capabilities;

} // namespace sycl::info::context
```

(See https://registry.khronos.org/SYCL/specs/sycl-2020/html/sycl-2020.html#appendix.context.descriptors)

I think there is a similar mistake for the platform information descriptors, which I missed when reviewing your previous commit.  Can you fix that also?

Reply:
Fixed with commit : https://github.com/KhronosGroup/SYCL_Reference/pull/73/commits/288589fcfa254916fce09085d3a5e5e3b171272c